### PR TITLE
Catching up with latest certified versions

### PR DESCRIPTION
**Pull request by bwised**
https://chartboost.atlassian.net/browse/HB-7689 "update 5.x.x.x.x partner adapter version to the latest"

### CODE REVIEW

File: .github/workflows/automated-create-release-version.yml
Comment (chauduyphanvu) [inline .github/workflows/automated-create-release-version.yml:1]: we don't need this file. it's already on `main` so when `feature/vc/mediation-5.0.0` is merged to `main` it'll have it

Comment (bwised) [reply .github/workflows/automated-create-release-version.yml:1]: Here are my steps:

```
git checkout main
git pull
git checkout feature/vc/mediation-5.0.0
git checkout -b adapter/rg/<whatever the version will be>
git merge main
<fix conflicts>
git push
```

Comment (bwised) [reply .github/workflows/automated-create-release-version.yml:1]: Should I instead just update the SDK version in this branch instead then?  Maybe that's the best way to go.

Comment (chauduyphanvu) [reply .github/workflows/automated-create-release-version.yml:1]: Yeah it's the `git merge main` that brings in the extra diff. I mean if you want to go that route, feel free to exclude changes to GitHub Actions files and just commit the other things.

**Comment by bwised** [comment]
I'll just close this PR I guess.  I'm not sure what the big deal is, but I'll just redo and instead manually pull in individual files into a new branch the annoying extremely tedious way instead of this harmless easy way.

**Pull request closed without merging**